tests/spark/test_spark_model_export.py

```diff
@@ -640,45 +640,58 @@ def test_model_is_recorded_when_using_direct_save(spark_model_iris):
 
 
 @pytest.mark.parametrize(
-    "artifact_uri, db_runtime_version,mlflowdbfs_disabled,mlflowdbfs_available,expectedURI",
+    "artifact_uri, db_runtime_version,mlflowdbfs_disabled,mlflowdbfs_available,dbutils_available,expectedURI",
```

Comment:
```suggestion
    (
      "artifact_uri",
      "db_runtime_version",
      "mlflowdbfs_disabled",
      "mlflowdbfs_available",
      "dbutils_available",
      "expectedURI",
    )
```

We can use `tuple` here.

Reply:
:D :D! Awesome! fixed, thanks Haru!